services/spar/src/Spar/Intra/BrigApp.hs

```diff
@@ -80,6 +80,7 @@ veidFromUserSSOId ::
   m ValidScimId
 veidFromUserSSOId ssoId mEmail = case ssoId of
   UserSSOId uref -> do
+    -- let eid = CI.original $ uref ^. SAML.uidSubject . to SAML.unsafeShowNameID
```

Comment:
yeah maybe we should do that.

see also: https://wearezeta.atlassian.net/browse/WPB-1506